packages/twenty-server/src/workspace/messaging/services/fetch-messages-by-batches.service.ts

```diff
@@ -197,6 +198,21 @@ export class FetchMessagesByBatchesService {
             ...this.formatAddressObjectAsParticipants(bcc, 'bcc'),
           ];
 
+          let textWithoutReplyQuotations = text;
+
+          if (text)
+            try {
+              textWithoutReplyQuotations = planer.extractFrom(
+                text,
+                'text/plain',
```

Comment:
Good to know we can do it with html too 👍 Let's do it when we actually use html since it needs an extra dependency as you told me.